Makefile

```diff
@@ -26,3 +26,7 @@ build: ## Build docker images
 
 bash-php: ## Open a bash in php container
 	docker-compose run php bash
+
+test-php: ## Launch phpunit tests
```

Comment:
Can you rename it to `test` or create a new make command that launch it?
This way we can just run test with `make test` just like all our projects.

Moreover, you can add e2e tests to the `make test` command in the future this way